packages/api-explorer/src/JsonForm/custom-bootstrap4.css

```diff
@@ -28,6 +28,10 @@
     margin: 0px;
 }
 
+.btn:focus, .btn-secondary:focus {
+    box-shadow: none !important;
```

Comment:
The important is not avoidable?

Reply:
Is avoidable in this way:
```
.btn:focus, 
.btn-secondary:focus, 
.btn-secondary:not(:disabled):not(.disabled):active:focus, 
.show>.btn-secondary.dropdown-toggle:focus
.btn-secondary.dropdown-toggle:focus {
    box-shadow: none;
}
```
Because are set box-shadow in a lot of different moment